Changelog.md

```diff
@@ -16,6 +16,9 @@ __New Features__
 - Allows heating/cooling seasons that don't span the entire year.
 - Allows calculating one or more utility bill scenarios (e.g., net metering vs feed-in tariff compensation types for a simulation with PV).
 - Allows setting the EnergyPlus temperature capacitance multiplier.
+- New lithium ion battery capabilities:
+  - Detailed charging/discharging schedules via CSV files.
+  - Allows modeling a scheduled battery without PV.  
```

Comment:
I guess technically this should say:
```suggestion
  - Allows modeling a scheduled battery (with or without PV).
```

Reply:
Or maybe it's not needed at all since the row above basically says the same thing?

Reply:
I'm trying to allude to the fact that _scheduled_ batteries (without PV) actually get modeled. It's probably safe to remove.

Reply:
I understood the intention -- but it might lead a user to think you can't model a scheduled battery w/ PV.